release_notes/374-release-notes.adoc

```diff
@@ -0,0 +1,321 @@
+
+.Release dates
+[options="header"]
+|====
+|{product-title-short} version |Released on
+|`3.74` |27 February 2023
+|====
+
+[id="about-this-release-374"]
+== About this release
+
+{product-title-short} 3.74 includes:
+
+* IBM Power and IBM Z support for secured clusters 
+* Clair scanner version 4 support
+* Network graph version 2.0 (Technology Preview)
+* Updated Global Search in the {product-title-short} portal 
+* Extra fields for improved syslog integration
+* Troubleshooting guide enhanced when Collector kernel module is missing
+* Scale and performance improvements
+* Features available if PostgreSQL database option (Technology Preview) is installed:
+** Collections
+** Policy categories
+* Feature enhancements and bug fixes
+
+[id="new-features-374"]
+== New features
+
+[id="ibm-power-z-374"]
+=== IBM Power and IBM Z support for secured clusters
+
+{product-title-short} version 3.74 extends support for {product-title-short} secured clusters for {osp} version 4.10, 4.11 and 4.12 to IBM Z and IBM Power architectures. With {product-title-short} version 3.74, you can install secured cluster services on {osp} on IBM Power and IBM Z by using the {product-title-short} Operator. 
+
+[NOTE]
+====
+* You can now secure IBM Power and IBM Z clusters with {product-title-short}.  Central is not supported.
+* The Collector is delivered as a kernel module and eBPF probe for IBM Power, but is only delivered as a kernel module for IBM Z.  Red Hat plans to add support for eBPF probes for IBM Z in a future release. 
+* Scanning is only supported for AMD64 images.
+====
+
+[id="clair-v4-374"]
+=== Clair scanner version 4 support
+
+Clair is a set of microservices that perform vulnerability scanning of container images associated with multiple Linux operating systems. If you are using Clair version 4 (v4), you can now integrate Clair v4 with {product-title-short} to get image vulnerability data. Integrating Clair v4 enables you to see image vulnerabilities from multiple sources in the {product-title-short} portal and triage them from one place.
+
+[NOTE]
+====
+* Red Hat has deprecated the previous CoreOS Clair integration in favor of Clair v4 integration. 
+* {product-title-short} version 4.0 will not support the link:https://quay.github.io/clair/concepts/authentication.html[JSON Web Token (JWT)-based authentication option] for Clair v4 integration.
+====
+
+[id="network-graph-2-374"]
+=== Network Graph 2.0 (Technology Preview)
+
+:FeatureName: Network Graph 2.0
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+{product-title-short} provides a Network Graph upgrade that provides intuitive navigation, enhanced functionality, and crisp graphs by using the Patternfly library. The new graph provides high-level and detailed information about deployments, network flows, and network policies in your environment. Highlights of the network graph include: 
```

Comment:
Struggled with this a bit because Network Graph is a menu item and a feature. So maybe I won't capitalize when it's referring to the feature, but will when it's used as the menu item....